_scss/_uswds-theme-custom-styles.scss

```diff
@@ -28,20 +28,78 @@ i.e.
 }
 
 .chp-home-hero {
-  @include u-bg($theme-color-chp-blue-light);
+  $hero-min-height-sm: 400px;
+  $hero-min-height-lg: 500px;
+  background-color: #000;
+  background-repeat: no-repeat;
+  min-height: $hero-min-height-sm;
+  background-image: url(../img/hero--home-sm.png);
+
+  @include at-media(desktop) {
+    background-position: right;
+    background-image: url(../img/hero--home-lg.png);
```

Comment:
Should we add a caption for the hero image?

> 1300 typists take civil service examinations in an indoor stadium of the McKinley High School in Washington DC on July 9, 1936.

OR

> 1300 typists take civil service examinations at McKinley High School in Washington, DC on July 9, 1936. Such examinations continued until 1981, when President Carter ended the practice. The Federal Government has not used examinations for hiring since... until now.